packages/editor/src/components/post-publish-panel/media-util.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * External dependencies
+ */
+import { v4 as uuid } from 'uuid';
+
+/**
+ * WordPress dependencies
+ */
+import { getFilename } from '@wordpress/url';
+
+// Generate a list of unique filenames given a list of URLs.
```

Comment:
Why no jsdoc? :-)

Reply:
Sorry, I'm still somewhat new to the codebase, and it didn't seem to be in widespread use 🙂